components/Home/sidebar/sidebar_items/PeerReviewSidebarItem.tsx

```diff
@@ -0,0 +1,132 @@
+import UserTooltip from "~/components/Tooltips/User/UserTooltip";
+import AuthorFacePile from "~/components/shared/AuthorFacePile";
+import VerifiedBadge from "~/components/Verification/VerifiedBadge";
+import StarInput from "~/components/Form/StarInput";
+import { getUrlToUniDoc } from "~/config/utils/routing";
+import { RHUser, UnifiedDocument } from "~/config/types/root_types";
+import Link from "next/link";
+import colors from "~/config/themes/colors";
+import { StyleSheet, css } from "aphrodite";
+
+const PeerReviewSidebarItem = ({
+  createdBy,
+  score,
+  unifiedDocument,
+}: {
+  createdBy: RHUser;
+  score: number;
+  unifiedDocument: UnifiedDocument;
+}) => {
+  const docTitle = unifiedDocument?.document?.title;
+  const url = getUrlToUniDoc(unifiedDocument) + "/reviews";
+
+  return (
+    <div className={css(styles.reviewSidebarItemContainer)}>
+      <Link style={{ textDecoration: "none" }} href={url}>
+        <div className={css(styles.reviewSidebarItem)}>
+          <div className={css(styles.reviewSidebarItemHeader)}>
+            <UserTooltip
+              createdBy={createdBy}
+              positions={["left"]}
+              targetContent={
+                <div style={{ display: "flex", alignItems: "center" }}>
+                  <AuthorFacePile
+                    authorProfiles={[createdBy.authorProfile]}
+                    withAuthorName
+                    horizontal
+                    margin={3}
+                    border={"none"}
+                    fontSize={14}
+                    imgSize={17}
+                  />
+                </div>
+              }
+            />
+            {createdBy.authorProfile.isVerified && (
+              <div style={{ marginLeft: 0, marginTop: 4 }}>
+                <VerifiedBadge height={18} width={18} />
+              </div>
+            )}
+            <span className={css(styles.reviewSidebarTitle)}>
+              <span>{"peer reviewed "}</span>
+            </span>
+          </div>
+          <div>
+            <StarInput value={score} readOnly size="small" />
+          </div>
+          <div className={css(styles.reviewSidebarItemContent)}>{docTitle}</div>
```

Comment:
nit: The bounties sidebar explicitly truncates titles longer than 180 characters ([source](https://github.com/ResearchHub/researchhub-web/blob/master/components/Home/sidebar/sidebar_items/BountiesSidebarItem.tsx#L83-L85)), we should probably do the same here.